### PR TITLE
www.minhaconexao.com.br

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/119314

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-23 в 19 10 43

</details><br/>

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Пропущен трекер
https://cdn.melhorplano.net/scripts/tracker/mpt.min.js
и для DNS
t.melhorplano.net

File: SpanishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ obutecodanet.ig.com.br#?#.elementor-section-wrap > .elementor-section:has(:is(di
 obutecodanet.ig.com.br#?#.elementor-section-wrap > .elementor-section .elementor-element-populated > .elementor-element.elementor-widget-html:has(> .elementor-widget-container > div[id^="ad_"][id*="mrec_"])
 animesonlinep.com#?##sidebarGeral > .sidebar:has(> #tituloSidebar > center:contains(Patrocinadores))
 shoptime.com.br,submarino.com.br#?#div[class^="aspect-ratio"] > div[id^="div-gpt-"]:upward(1)
+minhaconexao.com.br##body > section[class]:has(> div[class] > div[class] > div[id] > [style^="height:"])
```
Comment (zloyden) [inline SpanishFilter/sections/css_extended.txt:15]: I see only div elements, so it should be
```
minhaconexao.com.br##body > section[class^="c"]:has(> div[class] > div[class] > div[id] > div[style^="height:"])
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SpanishFilter/sections/css_extended.txt (hunk 1 above)
Comment (zloyden) [reply SpanishFilter/sections/css_extended.txt:15]: I've already added trackers, so just do a small fix.

**Review by zloyden — APPROVED** [review]

**Comment by Agronault** [comment]
👀

**Comment by JordanAmaralVicente** [comment]
👀 👎🏻

**Comment by Sergey-Lyapin** [comment]
FQM

**Pull request merged**